Alguns ajustes de front-end

lucasnantonio opened the pull request.

lucasnantonio committed:
Troca de marca da barra superior

Troca de Reboo pra Nossas Cidades

lucasnantonio committed:
Corrige largura do menu

lucasnantonio committed:
Melhora visibilidade do tipo de bloco escolhido

lucasnantonio committed:
Dá estilo aos botões que usam imagem no hover

Botões com imagem são ruins, mas com efeito no hover eles podem ficar
um pouco mais óbvios e clicáveis.

lpirola commented on app/styles/basscss_and_override.scss:
```diff
@@ -25,14 +25,32 @@ form .button { height: 3rem; }
+
+// DÁ EFEITO EM BOTÕES COM IMAGEM
+
+a img{
```
Definindo as propriedades dessa forma os elementos onde elas se aplicam são muito genéricos. É melhor especificar um pouco mais afim de evitar que elas se apliquem a elementos indesejados.

lucasnantonio replied:
Não sei como especificar mais...de fato eu quero que todas as imagens com link incorporado pelo usuário tenham um efeito de botão.

lpirola replied:
![](http://www.satirinhas.com/wp-content/uploads/2014/03/grandes-poderes-trazem-grandes-responsabilidades.jpg)

lucasnantonio replied:
hahahahahahahhhahahahahah genial

lucasnantonio committed:
Dá mais especificidade ao efeito de botão para imagens

Utilizando .widget a img:hover, somente as imagens dentro dos blocos de
texto (widget) serão afetadas. Que tal?

lpirola commented:
Maneiríssimo!

leticia commented on app/styles/basscss_and_override.scss:
```diff
@@ -25,14 +25,32 @@ form .button { height: 3rem; }
 	text-decoration:none;
 	border-left:3px solid #20BDE9;
+}
+.button..col.is-active{
```
Oops, there's a sparing dot here: `.button..col.is-active`

lucasnantonio committed:
Removes extra dot dot
